tests/misc/vhost.conf

```diff
@@ -28,12 +28,12 @@
   </Directory>
 
   # Wire up Apache to use Travis CI's php-fpm.
-  <IfModule mod_fastcgi.c>
-    AddHandler php5-fcgi .php
-    Action php5-fcgi /php5-fcgi
-    Alias /php5-fcgi /usr/lib/cgi-bin/php5-fcgi
-    FastCgiExternalServer /usr/lib/cgi-bin/php5-fcgi -idle-timeout 120 -host 127.0.0.1:9000 -pass-header Authorization
-  </IfModule>
+  # <IfModule mod_fastcgi.c>
```

Comment:
I suspect, travisCI does this php stuff slightly different. So maybe NOT extra installing php7.2-fpm from distro is a better choice. fcgid_module proxy_fcgid_module are still needed.

But then we somehow should figure out, where this travis php-fpm listens and let apache know the port via the vhost file.

Possibly:
```
  <FilesMatch ".+\.php$">
    SetHandler "proxy:fcgi://localhost:9000"
  </FilesMatch>
```
(Wild guessing again, though...)

Reply:
And, of course, without that IfModule wrapper as fastcgi isn't installed/enabled.